observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,387 @@
+     application:
+       namespaces:
+       - foo
+       - bar
+       includes:
+       - namespace: my-important
+         container: main
+       excludes:
+       - container: too-verbose
+----
+
+.6.0 Application Input with Namespace and Container Includes and Excludes
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+   - name: application-logs
+     type: application
+     application:
+       includes:
+       - namespace: foo
+       - namespace: bar
+       - namespace: my-important
+         container: main
+       excludes:
+       - container: too-verbose
+----
+
+[NOTE]
+====
+*application*, *infrastructure*, and *audit* are reserved words and cannot be used as names when defining an input.
+====
+
+=== Input Receivers
+
+Changes to input receivers include:
+
+* Explicit configuration of the type at the receiver level.
+* Port settings moved to the receiver level.
+
+.5.9 Input Receivers
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    receiver:
+      http:
+        port: 8443
+        format: kubeAPIAudit
+  - name: a-syslog
+    receiver:
+      type: syslog
+      syslog:
+        port: 9442
+----
+
+.6.0 Input Receivers
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+  inputs:
+  - name: an-http
+    type: receiver
+    receiver:
+      type: http
+      port: 8443
+      http:
+        format: kubeAPIAudit
+  - name: a-syslog
+    type: receiver
+    receiver:
+      type: syslog
+      port: 9442
+----
+
+== Output Specifications
+
+High-level changes to output specifications include:
+
+* URL settings moved to each output type specification.
+* Tuning parameters moved to each output type specification.
+* Separation of TLS configuration from authentication.
+* Explicit configuration of keys and secret/configmap for TLS and authentication.
+
+== Secrets and TLS Configuration
+
+Secrets and TLS configurations are now separated into authentication and TLS configuration for each output. They must be explicitly defined in the specification rather than relying on administrators to define secrets with recognized keys. Upgrading TLS and authorization configurations requires administrators to understand previously recognized keys to continue using existing secrets. Examples in the following sections provide details on how to configure *ClusterLogForwarder* secrets to forward to existing Red Hat managed log storage solutions.
+
+== Red Hat Managed Elasticsearch
+
+.v5.9 Forwarding to Red Hat Managed Elasticsearch
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  logStore:
+    type: elasticsearch
+----
+
+.v6.0 Forwarding to Red Hat Managed Elasticsearch
+[source,yaml]
+----
+apiVersion: "observability.openshift.io/v1"
```

Comment:
This example sounds so much realistic, something that I could copy, paste and work, but it won't do as not defined the serviceAccount, then, not able to read logs.

If it's not expected to be an real/functional example, then, I'd add to the end of the example something like "..." or I'd indicate explicitly